src/redux/Books/book.js

```diff
@@ -2,17 +2,38 @@ import { createSlice } from '@reduxjs/toolkit';
 
 const bookSlice = createSlice({
   name: 'books',
-  initialState: [],
+  initialState: [
+    {
+      item_id: 'item1',
+      title: 'The Great Gatsby',
+      author: 'John Smith',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item2',
+      title: 'Anna Karenina',
+      author: 'Leo Tolstoy',
+      category: 'Fiction',
+    },
+    {
+      item_id: 'item3',
+      title: 'The Selfish Gene',
+      author: 'Richard Dawkins',
+      category: 'Nonfiction',
+    },
+  ],
 
   reducers: {
     addBook: (state, action) => {
       const newBook = {
-        id: Date(),
+        id: action.payload.item_id,
         title: action.payload.title,
+        author: action.payload.author,
+        category: action.payload.category,
       };
       state.push(newBook);
     },
-    removeBook: (state, action) => state.filter((book) => book.id !== action.payload.id),
+    removeBook: (state, action) => state.filter((book) => book.key !== action.payload.id),
   },
```

Comment:
- [ ] Good job working on an event that removes a book from the `books` array, however, when we remove one book, all of them (books) us to be removed, we suppose to remove only one we need to remove. Kindly fix it 👍 😃